MyFirstSlideshow/app/src/main/res/layout/activity_main.xml

```diff
@@ -4,11 +4,87 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context="com.charles.myfirstslideshow.MainActivity">
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        />
+        <ImageView
+            android:layout_width="match_parent"
+            android:layout_height="400px"
+            android:id="@+id/image1"/>
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+            <View
+                android:id="@+id/dots1dot1"
+                android:layout_width="100px"
```

Comment:
activity_main : line 17 - 18 - when specifying the layout_weight in that case layout_width should be 0dp and height should be wrap_content

Reply:
activity_main : line 20 - follow the naming convention according to the use of that image in application.